packages/arb-token-bridge-ui/src/pages/_app.tsx

```diff
@@ -49,6 +50,27 @@ Sentry.init({
   }
 })
 
+if (
+  typeof window !== 'undefined' &&
+  typeof process.env.NEXT_PUBLIC_POSTHOG_KEY === 'string'
+) {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY, {
```

Comment:
_Sanity check_
This key will be exposed in the client. The reason why I think it's fine are the following:
1. The key is write only. No one can pull our analytics data with it.
2. Data capturing is restricted and has to come from the bridge. Otherwise it won't be pushed to our analytics (not even localhost). This is configured in the posthog account.

Please think of anything that could go wrong.

Reply:
Once we agree on this I will add the key to the build

Reply:
Yup this is fine, though it needs to be added on Vercel, not GitHub